stackslib/src/net/mod.rs

```diff
@@ -647,6 +647,7 @@ pub struct StacksNodeState<'a> {
     inner_mempool: Option<&'a mut MemPoolDB>,
     inner_rpc_args: Option<&'a RPCHandlerArgs<'a>>,
     relay_message: Option<StacksMessageType>,
+    ibd: bool,
```

Comment:
IMO 3-letter variable names in struct fields should usually have a doc comment clarifying what those letters mean (especially if type is just `bool`)

```suggestion
    /// Are we in Initial Block Download (IBD) phase?
    ibd: bool,
```